tls/s2n_handshake.h

```diff
@@ -53,6 +53,13 @@ struct s2n_handshake {
     struct s2n_hash_state sha256;
     struct s2n_hash_state sha384;
     struct s2n_hash_state sha512;
+    struct s2n_hash_state md5_sha1;
+
+    /* Used for SSLv3 */
+    struct s2n_hash_state sslv3_md5_copy;
```

Comment:
Where are these `sslv3_***_copy` Hash States used? Is that in a later commit? Same question for the `tls_hash_copy`.